corehq/motech/repeaters/optionvalue.py

```diff
@@ -65,3 +67,8 @@ def __set__(self, obj, value):
 @attr.s
 class OptionSchema:
     options = attr.ib()
+
+
+def _assert_options(obj):
+    assert hasattr(obj, 'options') and isinstance(obj.options, dict), \
```

Comment:
I considered duck-typing instead of using `isinstance()`. We use `setdefault()`, `__getitem__()` and `__setitem__()` so anything that implements those will work -- it doesn't need to subclass `dict`. But `dict` seemed more readable. I'm open to pushback.